Usgs/hazdev template#121

jmfee-usgs opened the pull request:
Fixes usgs/hazdev-template#121

- Support cooperator logos.
- Update print styles for banner to display site title

ehunter-usgs commented:
I'm wondering if it makes sense to define/build the contributor logo markup in config.inc.php, then contributors would just be a reference to the correct contributor key. This way we standardize the layout for each of our contributors?

I realize the tradeoff is that we then need to update the template each time we want to add a contributor, but it's just a thought that would promote consistency across our pages.

jmfee-usgs commented:
The contributors will be loaded from the documentation project, and added to the event page dynamically:
    http://usgs.github.io/anss-comcat/catalogs.json

The php template variable is for more static content, like a research project.  I'm open to suggestions for how php generates this content.  In other parts of the template we've moved towards directly embedded markup, `$HEAD` and `$FOOT` instead of `$STYLESHEETS` and `$SCRIPTS`, because it's more obvious what is output to the page.

ehunter-usgs commented:
Is ANSS the only logo that needs to be included in this project? All contributor logos will be added to the anss-comcat project?

jmfee-usgs commented:
Yes, all other logos will be in either the anss-comcat project, or as site content.
